src/style.css

```diff
@@ -1,3 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  --hover-color: #00acdc;
```

Comment:
Move to the :root selector

Reply:
Could you please elaborate a bit more.  Shall I move it to the body selector, instead? I was thinking that html - is the root selector